.github/workflows/main.yml

```diff
@@ -17,10 +17,10 @@ jobs:
             node-tag: v6.8.0 # v6.3.0 to v6.7.0 use a different config format
             compiler-tag: v7.4.0
             aux-ci-run: true
-          - node-version: 20
-            node-tag: latest
-            compiler-tag: latest
-            aux-ci-run: true
+          #- node-version: 20
```

Comment:
delete?

Reply:
no I'll keep it for now, will have the same message as here after rebase https://github.com/unit214/aeproject/pull/24/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R21